### ISSUE TITLE
[Feature] Integrate annotations

### ISSUE THREAD

**Issue by JamesGalt** [ilgizar/ilgizar-candlestick-panel#5]
It would be nice to have annotations in the panel
Thanks

**Comment by JamesGalt** [comment]
Hi, any idea on this one? 
Thanks

**Comment by ilgizar** [comment]
Only the transfer of the appropriate code from the main branch to the plugin. I remember and eventually integrate this feature.

**Comment by JamesGalt** [comment]
Thanks it's quite handy. ;)

**Comment by JamesGalt** [comment]
Hi,
I really need this, do you plan to add this feature ?

**Comment by ilgizar** [comment]
I agree that it is very convenient, but I can not promise anything in terms of time.
